prow/bugzilla/client_test.go

```diff
@@ -930,3 +930,156 @@ func TestGetExternalBugPRsOnBug(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAllClones(t *testing.T) {
+	exists := struct{}{}
+	testcases := map[int]struct {
+		bugDetails     string
+		prDetails      string
+		expectedClones map[int]struct{}
+	}{
+		1843407: {
+			`{"bugs" : [{ "alias" : [], "assigned_to" : "Sudha Ponnaganti", "assigned_to_detail" : {"email" : "sponnaga","id" : 426940,"name" : "sponnaga","real_name" : "Sudha Ponnaganti" }, "blocks" : [1844101 ], "cc" : ["aos-bugs@redhat.com","Eric Paris","Joanne Okerman","Roy Golan","Steve Kuznetsov","Sudha Ponnaganti" ], "cc_detail" : [{"email" : "aos-bugs","id" : 385833,"name" : "aos-bugs","real_name" : "aos-bugs@redhat.com"},{"email" : "eparis","id" : 165117,"name" : "eparis","real_name" : "Eric Paris"},{"email" : "jokerman","id" : 344626,"name" : "jokerman","real_name" : "Joanne Okerman"},{"email" : "rgolan","id" : 300276,"name" : "rgolan","real_name" : "Roy Golan"},{"email" : "skuznets","id" : 381851,"name" : "skuznets","real_name" : "Steve Kuznetsov"},{"email" : "sponnaga","id" : 426940,"name" : "sponnaga","real_name" : "Sudha Ponnaganti"} ], "cf_clone_of" : null, "cf_doc_type" : "If docs needed, set a value", "cf_environment" : "", "cf_fixed_in" : "", "cf_last_closed" : null, "cf_release_notes" : "", "cf_target_upstream_version" : "", "classification" : "Red Hat", "component" : ["Unknown" ], "creation_time" : "2020-06-03T09:03:41Z", "creator" : "Gal Zaidman", "creator_detail" : {"email" : "gzaidman","id" : 422274,"name" : "gzaidman","real_name" : "Gal Zaidman" }, "deadline" : null, "depends_on" : [], "docs_contact" : "", "dupe_of" : null, "groups" : [], "id" : 1843407, "is_cc_accessible" : true, "is_confirmed" : true, "is_creator_accessible" : true, "is_open" : true, "keywords" : [], "last_change_time" : "2020-06-07T11:08:27Z", "op_sys" : "Unspecified", "platform" : "Unspecified", "priority" : "unspecified", "product" : "OpenShift Container Platform", "qa_contact" : "Jianwei Hou", "qa_contact_detail" : {"email" : "jhou","id" : 332453,"name" : "jhou","real_name" : "Jianwei Hou" }, "resolution" : "", "see_also" : [], "severity" : "medium", "status" : "VERIFIED", "summary" : "[oVirt] add oVirt as a provide to openshift tests", "target_milestone" : "---", "target_release" : ["4.6.0" ], "url" : "", "version" : ["4.5" ], "whiteboard" : ""}],"faults" : []}`,
```

Comment:
Please do not put real email addresses and names on github, even if they can be retrieved by anyone elsewhere

Reply:
ok, i'll black that out